### PR TITLE
Bugfix/ae 2511/dj/generic

### PR DESCRIPTION
**Pull request by jesse1d**
Minor corrections following processing of 46 download files.

### CODE REVIEW

File: analysis_engine/plot_flight.py [hunk 1]
```diff
@@ -213,10 +213,13 @@ def track_to_kml(hdf_path, kti_list, kpv_list, approach_list,
         # Get altitude param from hdf.
         if plot_altitude and plot_altitude in hdf.keys():
             alt = derived_param_from_hdf(hdf[plot_altitude]).get_aligned(one_hz)
+            if np.ma.count(alt.array):
```
Comment (kirku1g) [inline analysis_engine/plot_flight.py]: The lines following this if statement should be indented.

File: analysis_engine/plot_flight.py [hunk 2]
```diff
@@ -213,10 +213,13 @@ def track_to_kml(hdf_path, kti_list, kpv_list, approach_list,
...
             alt.array = repair_mask(alt.array, frequency=alt.frequency, repair_duration=None)
             alt.array = ut.convert(alt.array, ut.FT, ut.METER)
         else:
             alt = None
+        else:
```
Comment (kirku1g) [inline analysis_engine/plot_flight.py]: This else statement is duplicated.

File: analysis_engine/plot_flight.py (hunk 1 above)
Comment (jesse1d) [reply analysis_engine/plot_flight.py]: Should read:
        if plot_altitude and plot_altitude in hdf.keys():
            alt = derived_param_from_hdf(hdf[plot_altitude]).get_aligned(one_hz)
            if np.ma.count(alt.array):
                alt.array = repair_mask(alt.array, frequency=alt.frequency, repair_duration=None)
                alt.array = ut.convert(alt.array, ut.FT, ut.METER)
        else:
            alt = None

File: analysis_engine/plot_flight.py (hunk 2 above)
Comment (jesse1d) [reply analysis_engine/plot_flight.py]: Agreed. As above.

**Pull request merged**